middleware/transaction/source/manager/admin/server.cpp

```diff
@@ -59,13 +60,20 @@ namespace casual
             {
                return { {
                      { service::name::state,
-                        std::bind( &local::state, std::placeholders::_1, std::ref( state)),
+                        local::state( state),
+                        common::service::transaction::Type::none,
+                        common::service::visibility::Type::undiscoverable,
+                        common::service::category::admin
+                     },
+                     { service::name::scale::resource::proxies,
+                        local::scale::resource::proxy::instances( state),
                         common::service::transaction::Type::none,
                         common::service::visibility::Type::undiscoverable,
                         common::service::category::admin
                      },
-                     { service::name::scale::instances,
-                        std::bind( &local::scale::instances, std::placeholders::_1, std::ref( state)),
+                     // deprecated
+                     { ".casual/transaction/scale/instances",
+                        local::scale::resource::proxy::instances( state),
                         common::service::transaction::Type::none,
                         common::service::visibility::Type::undiscoverable,
                         common::service::category::admin
```

Comment:
Change to `common::service::category::deprecated` ?

Reply:
Hmm, this would probably expose the service during `casual service --list-services`. I think we just partition all services on predicate `service.category == .admin`  -> [services, admin-services]. 
Maybe we should use predicate `services.category.starts_with( '.')` ? Sorta same thing as hidden unix files/directorys.